README copy.md

```diff
@@ -0,0 +1,131 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microcerse] <a name="about-project"></a>
+
+**[Leadersboard]**
+
+> "Leadersboard" is a tool that helps to organize Scores. It simply lists the scores that students has score in a test. In this particular repo, We are building this Leaderboard with using HTML, CSS, JavaScript (ES6), and Webpack.
+
+## 🛠 Built With <a name="built-with"></a>
+- [x] HTML, CSS and JavaScript
+- [x] Linters: Lighthouse, Webhint, Stylelint and ESLint
+- [x] VS Code
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [x] A web browser like Google Chrome
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+### Setup
+
+Clone this repository to your desired folder using the link given below:
+
+use the git clone command to clone this repository or download ZIP
+
+### Install
+
+Install this project with:
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/Odongo006/Leadersboard.git
+```
+### Usage
+
+To run the project, execute the following command: 
+- Repository[@repo](https://github.com/Odongo006/Leadersboard.git)
+
+
+
+
+
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+### Key Features
+- Webpack
+- Linters
+
+### Future Features
+
+- Babel
+- React
```

Comment:
- [Optional] Please we are talking of app features(What the app can accomplish like adding a score to the board)